connector/spanmetricsconnector/connector.go

```diff
@@ -520,11 +522,27 @@ func (p *connectorImp) buildAttributes(serviceName string, span ptrace.Span, res
 	if !contains(p.config.ExcludeDimensions, statusCodeKey) {
 		attr.PutStr(statusCodeKey, traceutil.StatusCodeStr(span.Status().Code()))
 	}
+
 	for _, d := range dimensions {
 		if v, ok := utilattri.GetDimensionValue(d, span.Attributes(), resourceAttrs); ok {
 			v.CopyTo(attr.PutEmpty(d.Name))
 		}
 	}
+
+	if !contains(p.config.ExcludeInstrumentationScope, instrumentationScopeNameKey) && instrumentationScope.Name() != "" {
+		attr.PutStr(instrumentationScopeNameKey, instrumentationScope.Name())
+	}
+	if !contains(p.config.ExcludeInstrumentationScope, instrumentationScopeVersionKey) && instrumentationScope.Version() != "" {
```

Comment:
As the changelod doc described, we only need check the `instrumentationScope.Name()`.

Reply:
Ok, I will remove the version them.